### DISCUSSION TITLE
How much latency is normal when using Realtime?

### DISCUSSION THREAD

**Discussion by SSA4** [voicepaw/so-vits-svc-fork#184] [Q&A] [answered]
Hi,
I installed "so-vits-svc-fork"  succesfully yesterday and it works very well on my system. But I recognized thats there is a big amount of latency on my PC if I using the Realtime function. I using a Focusrite scarlett 2i2 audio interface with ASIO but the latency is still very high. Even if I activate "Passthrough original audio" my original voice has this latency. Is this normal or do I something wrong? My CPU is AMD Ryzen 9 5900X and my GPU is NVIDIA RTX 3090 Ti.

**Comment by 34j** [comment, marked as answer]
Passthrough - is not the actual one and it should be crossfade + block + additional_after.
